forum/qa-include/pages/message.php

```diff
@@ -122,23 +122,30 @@
 					$messageid = null;
 
 				$canreply = !(qa_get_logged_in_flags() & QA_USER_FLAGS_NO_MESSAGES);
-
-				$more = strtr(qa_lang($canreply ? 'emails/private_message_reply' : 'emails/private_message_info'), array(
-					'^f_handle' => $fromhandle,
-					'^url' => qa_path_absolute($canreply ? ('message/'.$fromhandle) : ('user/'.$fromhandle)),
-				));
-
-				$subs = array(
-					'^message' => $inmessage,
-					'^f_handle' => $fromhandle,
-					'^f_url' => qa_path_absolute('user/'.$fromhandle),
-					'^more' => $more,
-					'^a_url' => qa_path_absolute('account'),
-				);
-
-				if (qa_send_notification($toaccount['userid'], $toaccount['email'], $toaccount['handle'],
-						qa_lang('emails/private_message_subject'), qa_lang('emails/private_message_body'), $subs))
-					$messagesent = true;
+				if ($toaccount['pwemail'] == 1) {

```

Comment:
To kod Q2A, ja tego nie pisałem a jedynie dodałem warunek - jakby więc poprawiać cały ich kod to na pewno wiele rzeczy można zrobić bardziej elegancko.